var/spack/repos/builtin/packages/faodel/package.py

```diff
@@ -16,28 +16,29 @@ class Faodel(CMakePackage):
     maintainers = ['tkordenbrock', 'craigulmer']
 
     version('master', branch='master')
+    version('1.2108.1-rc1', tag='v1.2108.1-rc1')
```

Comment:
Thanks for the clarification.  Earlier this year, we had some problems with our package definition and I wanted to make sure this would go through CI before we rolled the final release of v1.2108.1.  I planned to update the package definition when Faodel went final (sha256 vs tag, etc), then remove the WIP.  Is there a better way to do this?  In any case, I should have put a comment to this effect in the description.

Reply:
Ah, that makes sense. No worries. I'll mark this PR as a draft.